pandas/tests/resample/test_resample_api.py

```diff
@@ -14,10 +14,15 @@
                  end=datetime(2005, 1, 10), freq='Min')
 
 test_series = Series(np.random.rand(len(dti)), dti)
-test_frame = DataFrame(
+_test_frame = DataFrame(
```

Comment:
can u just inline this in the fixture?

can u fixturize the series as well

not so sure these are great names either

Reply:
Fixturing this kind of thing is like pulling my own teeth.  Closing.

Reply:
ok i can’t imagine it taking very long